.changeset/breezy-eyes-teach.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": patch
+---
+
+Fix history navigation to hash fragment on Firefox when using view transitions
```

Comment:
```suggestion
Fixes issues with navigation history in Firefox when attempting to navigate back to fragment links (e.g. `#about`) when using view transitions
```
It is indeed "fixes" (aka. "This PR.... fixes!) but I'd go a step further and say you could describe this in a tiny bit more user-friendly way! (Make it correct, but I'd go with a style like this)